Helper/FragmentHelper.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Helper;
+
+use Sonata\ArticleBundle\FragmentService\FragmentServiceInterface;
+use Sonata\ArticleBundle\Model\FragmentInterface;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Class FragmentHelper.
```

Comment:
... which I did not review. Do try to copy existing style, but don't assume existing code is perfect. This comment adds zero information, it's just noise,  so please improve or just remove it.